doc/manual/src/glossary.md

```diff
@@ -115,14 +115,14 @@
     then be built.
 
   - [reference]{#gloss-reference}\
-    A store path `P` is said to have a reference to a store path `Q` if
-    the store object at `P` contains the path `Q` somewhere. The
-    *references* of a store path are the set of store paths to which it
-    has a reference.
+    A [store object] `O` is said to have a *reference* to a store object `P` if the [store path] of `P` appears in the contents of `O`.
```

Comment:
This is not quite correct. References are not store objects but store paths.

There is no such thing as "the store path of P" since store objects don't have a uniquely determined store path. (E.g. there can be multiple store paths with the same contents.)